src/azure-map/README.md

```diff
@@ -78,10 +78,12 @@ mergeDataPoint(data) {
 Our AtlasMapComponent will create your map canvas, and add your Map Elements to map through added pins on it.<br>
 Also can added PopUps on pins through atlas-popup directive and ng-template in your parent component html file:
 ```html
+<am-map>
```

Comment:
In examples we should show working examples and this example does not work. `ng-template` is only applied when there are points on the map, but they are not provided here.